openscenario/openscenario_interpreter/include/openscenario_interpreter/procedure.hpp

```diff
@@ -106,15 +106,13 @@ STRIP_OPTIONAL(getTimeHeadway, std::numeric_limits<value_type>::quiet_NaN());
 
 FORWARD_TO_SIMULATION_API(attachDetectionSensor);
 FORWARD_TO_SIMULATION_API(attachLidarSensor);
-FORWARD_TO_SIMULATION_API(despawn);
 FORWARD_TO_SIMULATION_API(engage);
 FORWARD_TO_SIMULATION_API(getCurrentTime);
 FORWARD_TO_SIMULATION_API(initialize);
 FORWARD_TO_SIMULATION_API(isInLanelet);
 FORWARD_TO_SIMULATION_API(ready);
-FORWARD_TO_SIMULATION_API(setEntityStatus);
+FORWARD_TO_SIMULATION_API(setEntityStatus);  // XXX DEPRECATED
```

Comment:
Please use @deprecated in doxygen and deprecated pragma.

https://en.cppreference.com/w/cpp/language/attributes/deprecated